docs/visualization/visualize_unit_responses.ipynb

```diff
@@ -86,28 +86,24 @@
      "name": "stderr",
```

Comment:
<p>I think the institute generally tries to say "putative" neuron. It implies more certainty and scientific validity than hypothetical</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>